packages/core/src/lib/toast/__tests__/toast.spec.ts

```diff
@@ -42,7 +41,6 @@ describe('toast', () => {
     await act(() => {
       context.toast({
         message: 'This is a success toast message',
-        variant: ToastVariant.Success,
```

Comment:
I think it's worth keeping this in the test, because otherwise you don't have a test that uses the explicit `variant` option. It will also fix your lint issue